posthog/decorators.py

```diff
@@ -44,9 +48,20 @@ def wrapper(self, request) -> T:
         # return cached result when possible
         if not should_refresh(request):
             cached_result_package = get_safe_cache(cache_key)
+
+            # ignore the bare exception warning. we never want this to fail
+            # noinspection PyBroadException
+            try:
+                path = path_tag_pattern.sub("", request.path)
```

Comment:
We do a similar thing in `CHQueries` middleware: Let's be consistent and not create a new patten.

`resolve(request.path).route`

Reply:
Ooh! #TIL

That's much nicer!